.github/workflows/build.yml

```diff
@@ -39,7 +41,7 @@ jobs:
           )
           echo "include=$MATRIX" | tee -a $GITHUB_OUTPUT
         env:
-          CIBW_ARCHS_LINUX: x86_64 aarch64
+          CIBW_ARCHS_LINUX: x86_64
```

Comment:
nit: should we remove this from here and move --archs x86_64 into the cibuildwheel command above?